discovery-infra/ocp.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import argparse
+import day2
+import utils
+import oc_utils
+import assisted_service_api
+import consts
+import sys
+
+
+def get_ocp_cluster(args):
+    if not args.cluster_id:
+        cluster_name = f'{args.cluster_name or consts.CLUSTER_PREFIX}-{args.namespace}'
+        tf_folder = utils.get_tf_folder(cluster_name, args.namespace)
+        args.cluster_id = utils.get_tfvars(tf_folder)['cluster_inventory_id']
+    client = assisted_service_api.create_client(
+            url=utils.get_assisted_service_url_by_args(args=args)
+    )
+    return client.cluster_get(cluster_id=args.cluster_id)
+
+def main(args):
+    if args.config_etc_hosts:
+        ocp_cluster = get_ocp_cluster(args)
+        ocp_cluster_name = ocp_cluster.name
+        ocp_api_vip_dnsname = "api." + ocp_cluster_name + "." + ocp_cluster.base_dns_domain
+        day2.config_etc_hosts(ocp_cluster.api_vip, ocp_api_vip_dnsname)
+    if args.get_cluster_api_vip:
+        ocp_cluster = get_ocp_cluster(args)
+        sys.stdout.write(ocp_cluster.api_vip)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="OCP")
+    parser.add_argument(
+        "-id", "--cluster-id",
+        help="Cluster id to install",
+        type=str,
+        default=None
```

Comment:
don't think you need to explicitly add `default=None `

Reply:
for sure? copied it from: https://github.com/openshift/assisted-test-infra/blob/a425c840c727b4e3be1c2341e8b70930b10cb216/discovery-infra/install_cluster.py#L158
maybe it's defaulted to an empty string if not specified? I'll give it a try